### DISCUSSION TITLE
Audiogridder crashing  on loading East West Play

### DISCUSSION THREAD

**Discussion by gatd** [apohl79/audiogridder#937] [Help]

**Comment by gatd** [comment]
AudioGridder is crashing on loading East West Play libraries

**Comment by Fittkenny** [comment]
maybe they are pirated?

**Comment by gatd** [comment]
they work fine in stand alone?

**Comment by gatd** [comment]
OK now fixed it by turning off sandboxing in preferences.

**Comment by gatd** [comment]
no its still crashing AG?
